charts/tempo/templates/networkpolicy.yaml

```diff
@@ -52,7 +52,7 @@ spec:
         - namespaceSelector:
             {{- toYaml . | nindent 12 }}
         {{- end }}
-        - podSelector:
+          podSelector:
```

Comment:
What would impact if above L51 explicitNamespacesSelector is empty? Would that break anything?

Reply:
@zanhsieh if `explicitNamespacesSelector` is empty, it wouldn't break anything since the code goes with `{{ with }}` in line 51, I tested this Network Policy previously by keeping `explicitNamespacesSelector` empty only, it didn't break anything and was working as expected.